src/pages/Donations/useDonations.ts

```diff
@@ -19,7 +20,7 @@ export default function useDonations() {
 
   const [params, setParams] = useState<DonationsQueryParams>({
     id: address || "",
-    chain_id: "80001",
+    chain_id: chainIds.polygon,
```

Comment:
We need to detect the chain that the user's wallet is connected to here, since they (unlike the Endowments who only receive on Polygon) can send donations from other chains.

Reply:
understood, thanks

Reply:
@abbasali1994 What's the status on this? Thanks!

Reply:
I have debugged this and found the following:
this param chain_id when set to polygon returns all the donations even if not of polygon. the filter tag within the donation page is responsible for filtering of donations as per selected chain in filter dropdown , the following is not connected with wallet in any sense (The address also is picked from query param and not from connected wallet)

If we try to make it such that it only returns donations of the connected network on first render it might become confusing for end user as the filters section also provides a network select option which would conflict with the network we select in wallet section.

Modifying "80001" -> chainIds.polygon should take care of test and prod deployment queries